src/components/cards/card-full/card-full.ts

```diff
@@ -0,0 +1,148 @@
+    const productPriceHolder = new BaseComponent('div', {
+      className: 'card__product_price_holder',
+    });
+    heartPrice.append(productPriceHolder);
+
+    const productPrice = new BaseComponent('span', { className: 'card__product_price' });
+    productPrice.setContent(`${this.data.price}$`);
+    productPriceHolder.append(productPrice);
+
+    this.heartToCart = new BaseComponent('span', { className: 'card__heart' });
+    heartPrice.append(this.heartToCart);
+    this.heartToCart.getNode().addEventListener('click', () => {
+      this.heartToCart.toggleClass('active');
+    });
+
+    const productImage = new BaseComponent('span', { className: 'card__product_image' });
+    productImage.getNode().style.backgroundImage = `url('${location.origin}/${this.data.pics[0]}')`;
+    cardFrame.append(productImage);
+
+    this.detailsLink = new Link({
+      href: NameRoute.Product.replace(':id', this.data.id),
+      className: 'details_link',
+    });
+    cardFrame.append(this.detailsLink);
+
+    const productDetails = new BaseComponent('span', { className: 'card__details' });
+    productDetails.setContent('Details');
+    this.detailsLink.append(productDetails);
+
+    const cardTitle = new BaseComponent('div', { className: 'card__title' });
+    this.append(cardTitle);
+
+    const brandName = new BaseComponent('div', { className: 'card__title brand_name' });
+    brandName.setContent(`${this.data.brandName}`);
+    cardTitle.append(brandName);
+
+    const productName = new BaseComponent('div', { className: 'card__title product_name' });
+    productName.setContent(`${this.data.productName}`);
+    cardTitle.append(productName);
+
+    const buttonsHolder = new BaseComponent('div', { className: 'card__buttons_holder' });
+    this.append(buttonsHolder);
+
+    const cardButtonAmount = new BaseComponent('div', {
+      className: 'card__buttons_holder button amount',
+    });
+    buttonsHolder.append(cardButtonAmount);
+
+    const minusItem = new BaseComponent('span', { className: 'minus' });
+    minusItem.setContent('-');
+    cardButtonAmount.append(minusItem);
+
+    this.amountOfAddedProduct = new BaseComponent('span', {
+      className: 'added_number',
+    });
+
+    this.amountOfAddedProduct.setContent(`${this.counter}`);
+    cardButtonAmount.append(this.amountOfAddedProduct);
+
+    const plusItem = new BaseComponent('span', { className: 'plus' });
+    plusItem.setContent('+');
+    cardButtonAmount.append(plusItem);
+
+    plusItem.getNode().addEventListener('click', () => {
+      if (this.counter < this.data.availableAmount) {
+        if (this.counter === 0) {
+          this.cardButtonAddToCart.setContent('Remove');
+        }
+        this.counter++;
+        this.amountOfAddedProduct.setInnerHTML(this.counter.toString());
+        this.cardPrice.setContent(`${this.counter * this.data.price}$`);
+        CartService.addToCart(this.data);
+      }
+    });
+
+    minusItem.getNode().addEventListener('click', () => {
```

Comment:
We should create a component, because + and - buttons are the same components, pass a content and callback with logic.